CMake/FindArmadillo.cmake

```diff
@@ -40,27 +40,14 @@ if(ARMADILLO_INCLUDE_DIR)
   # ------------------------------------------------------------------------
   #  Extract version information from <armadillo>
   # ------------------------------------------------------------------------
-
-  # WARNING: Early releases of Armadillo didn't have the arma_version.hpp file.
-  # (e.g. v.0.9.8-1 in ubuntu maverick packages (2001-03-15))
-  # If the file is missing, set all values to 0
-  set(ARMADILLO_VERSION_MAJOR 0)
-  set(ARMADILLO_VERSION_MINOR 0)
-  set(ARMADILLO_VERSION_PATCH 0)
-  set(ARMADILLO_VERSION_NAME "EARLY RELEASE")
```

Comment:
That's fair! You may even consider pushing upstream to cmake's own findArmadillo -- https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindArmadillo.cmake

Reply:
Perfect, I will do it when we are sure of these changes

Reply:
Up to you if you want to push this upstream; it might be useful to not strip functionality upstream though, since although we can say that we know that 8.400.0 or newer will be the minimum that we get, it may not be true for all uses.  That said, I have no idea who still has any Armadillo < 1 tarball anywhere, so perhaps it doesn't hurt for this change to submit it upstream. :)

Reply:
@rcurtin Sure, Armadillo < 1 is 20 years old I think

Reply:
Actually I think that's a typo---probably the date should be 2010-03-15. :)